src/components/SectionHeader/SectionHeader.scss

```diff
@@ -0,0 +1,35 @@
+@import '~scss/Variables';
+
+.section-header {
+  padding: 0px;
+}
+
+.section-header {
+  height: 300px;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .floating-shapes {
```

Comment:
I know you borrowed my code but I got lazy and did a bad thing here. It'd be better to wrap `<FloatingShapes ... />` in a separate `<div className='floating-shapes-wrapper'>...</div>` and use `.floating-shapes-wrapper` here instead because in its current state, it makes an assumption about the class name of a React component. The assumption is correct at the moment but if that changes at any point, the layout would break and it wouldn't be entirely clear as to why.